### PR TITLE
Restricting_s3access_by_VPC_endpoint

### PR DESCRIPTION
**Pull request by pvisweswar**
Ticket:- https://dimagi-dev.atlassian.net/browse/SAAS-12474

##### ENVIRONMENTS AFFECTED
staging

### CODE REVIEW

File: src/commcare_cloud/terraform/modules/network/main.tf [hunk 1]
```diff
@@ -559,3 +559,14 @@ resource "aws_security_group" "elasticache" {
     Name = "elasticache-${var.env}"
   }
 }
+
+resource "aws_vpc_endpoint" "s3" {
+  vpc_id       = aws_vpc.main.id
+  service_name = "com.amazonaws.us-east-1.s3"
+}
+
+
```
Comment (prem-fissionhq) [inline src/commcare_cloud/terraform/modules/network/main.tf:568]: remove extra space here.

File: src/commcare_cloud/terraform/modules/network/main.tf [hunk 2]
```diff
@@ -559,3 +559,14 @@ resource "aws_security_group" "elasticache" {
...
+resource "aws_vpc_endpoint_route_table_association" "private_s3" {
+  vpc_endpoint_id = aws_vpc_endpoint.s3.id
+  route_table_id  = aws_route_table.private.id
+}
```
Comment (prem-fissionhq) [inline src/commcare_cloud/terraform/modules/network/main.tf:572]: add space at the end of the line.

File: src/commcare_cloud/terraform/modules/network/main.tf (hunk 1 above)
Comment (pvisweswar) [reply src/commcare_cloud/terraform/modules/network/main.tf:568]: Ok sure Prem

File: src/commcare_cloud/terraform/modules/network/main.tf (hunk 2 above)
Comment (pvisweswar) [reply src/commcare_cloud/terraform/modules/network/main.tf:572]: Ok Sure Prem

**Comment by prem-fissionhq** [comment]
Duplicate Ticket: https://github.com/dimagi/commcare-cloud/pull/5102
@pvisweswar please close this

**Pull request closed without merging**